internal/v1/server_test.go

```diff
@@ -202,10 +202,40 @@ func TestWithoutOsbuildComposerBackend(t *testing.T) {
 			ArchitectureItem{
 				Arch:       "x86_64",
 				ImageTypes: []string{"aws", "gcp", "azure", "ami", "vhd"},
+				Repositories: []Repository{
+					{
+						Baseurl: common.StringToPtr("http://mirror.centos.org/centos/8-stream/BaseOS/x86_64/os/"),
+						Rhsm:    false,
+					}, {
+						Baseurl: common.StringToPtr("http://mirror.centos.org/centos/8-stream/AppStream/x86_64/os/"),
+						Rhsm:    false,
+					}, {
+						Baseurl: common.StringToPtr("http://mirror.centos.org/centos/8-stream/extras/x86_64/os/"),
+						Rhsm:    false,
+					}, {
+						Baseurl: common.StringToPtr("https://packages.cloud.google.com/yum/repos/google-compute-engine-el8-x86_64-stable"),
+						Rhsm:    false,
+					}, {
+						Baseurl: common.StringToPtr("https://packages.cloud.google.com/yum/repos/cloud-sdk-el8-x86_64"),
+						Rhsm:    false,
+					},
```

Comment:
I think it's best that we only return repositories which are valid for all image types (so which don't have `image_type_tags`), we can potentially add a query parameter in future which allows you to query the repos for a specific image type.

Reply:
Got it! Updated, now it returns only repositories that have no image type tags :)